Libraries/LibGfx/ClassicWindowTheme.h

```diff
@@ -26,8 +26,10 @@
 
 #pragma once
 
-#include <LibGfx/WindowTheme.h>
+#include <AK/NonnullOwnPtrVector.h>
 #include <LibGfx/Color.h>
+#include <LibGfx/WindowTheme.h>
+#include <Services/WindowServer/Button.h>
```

Comment:
This isn't right. WindowServer is a separate program and LibGfx should not depend on it.
Let's find a way to represent the necessary data in a LibGfx data structure instead :)

Reply:
Apologies for the brainletism. Second attempt :^)